lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -2622,6 +2622,31 @@ def model_find_objects(hash_of_objects, search_criteria, capacity = nil, date =
       end
     end
 
+    # If volume was specified, narrow down the matching objects
+    unless volume.nil?
+      # Skip objects that don't have fields for minimum_storage and maximum_storage
+      matching_objects = matching_objects.reject { |object| !object.key?('minimum_storage') || !object.key?('maximum_storage') }
+
+      # Skip objects that don't have values specified for minimum_storage and maximum_storage
+      matching_objects = matching_objects.reject { |object| object['minimum_storage'].nil? || object['maximum_storage'].nil? }
+
+      # Round up if volume is an integer
+      if volume == volume.round
+        volume += (volume * 0.01)
+      end
```

Comment:
This seems a hacky way to get decimal points for volumes. It sort of make sense for capacity data since the maximum ends in .99, but in the water heater storage, some maximum ends with integer.
What I worried about is if the volume = 120, then this code will cause no record found.
I suggest either move this code or update the water heater data.